### ISSUE TITLE
IMAP error: Error moving message UID 15516: copy failed: Error in IMAP command received by server

### ISSUE THREAD

**Issue by martinboller** [domainaware/parsedmarc#103]
With 6.5.0 and 6.5.1 I get the following errors (with debug)
`2019-07-27 16:33:29,411 - DEBUG - [__init__.py:996] - Processing message 1 of 6: UID 15515
2019-07-27 16:33:29,514 - DEBUG - [__init__.py:996] - Processing message 2 of 6: UID 15516
2019-07-27 16:33:29,616 - DEBUG - [__init__.py:996] - Processing message 3 of 6: UID 15517
2019-07-27 16:33:29,716 - DEBUG - [__init__.py:996] - Processing message 4 of 6: UID 15518
2019-07-27 16:33:29,800 - DEBUG - [__init__.py:996] - Processing message 5 of 6: UID 15519
2019-07-27 16:33:29,858 - DEBUG - [__init__.py:996] - Processing message 6 of 6: UID 15520
2019-07-27 16:33:29,964 - DEBUG - [__init__.py:1050] - Moving aggregate report messages from Inbox.DMARC to Inbox.Archive/Aggregate
2019-07-27 16:33:29,964 - DEBUG - [__init__.py:1056] - Moving message 1 of 6: UID 15515
2019-07-27 16:33:29,981 - ERROR - [__init__.py:1063] - IMAP error: Error moving message UID 15515: copy failed: Error in IMAP command received by server.
2019-07-27 16:33:29,981 - DEBUG - [__init__.py:1056] - Moving message 2 of 6: UID 15516
2019-07-27 16:33:29,997 - ERROR - [__init__.py:1063] - IMAP error: Error moving message UID 15516: copy failed: Error in IMAP command received by server.
2019-07-27 16:33:29,997 - DEBUG - [__init__.py:1056] - Moving message 3 of 6: UID 15517
2019-07-27 16:33:30,014 - ERROR - [__init__.py:1063] - IMAP error: Error moving message UID 15517: copy failed: Error in IMAP command received by server.
2019-07-27 16:33:30,014 - DEBUG - [__init__.py:1056] - Moving message 4 of 6: UID 15518
2019-07-27 16:33:30,030 - ERROR - [__init__.py:1063] - IMAP error: Error moving message UID 15518: copy failed: Error in IMAP command received by server.
2019-07-27 16:33:30,031 - DEBUG - [__init__.py:1056] - Moving message 5 of 6: UID 15519
2019-07-27 16:33:30,048 - ERROR - [__init__.py:1063] - IMAP error: Error moving message UID 15519: copy failed: Error in IMAP command received by server.
2019-07-27 16:33:30,049 - DEBUG - [__init__.py:1056] - Moving message 6 of 6: UID 15520
2019-07-27 16:33:30,066 - ERROR - [__init__.py:1063] - IMAP error: Error moving message UID 15520: copy failed: Error in IMAP command received by server.
`
Rolling back to 6.4.1 or 6.4.2 and everything works perfectly again.

**Comment by seanthegeek** [comment]
Hi,

It looks like I broke something when I refactored the mail functions into mailsuite. I need to reproduce the issue to be able to debug it. 

Can you set up a mailbox for me and put some messages in it so I can debug the script on your server?

If so, send creds to seanthegeek@pm.me.

**Comment by martinboller** [comment]
Mail sent, thanks.

/Martin

**Comment by seanthegeek** [comment]
I just fixed this in the latest development version, 6.5.2, but I'm going to try to wait to do an actual release until I can verify that I have also fixed issue #102, which I suspect is related.
 
In the meantime, you can upgrade to the latest dev release by running 

```
sudo -H pip3 install -U git+https://github.com/domainaware/parsedmarc.git
```

**Comment by martinboller** [comment]
On Tue, 2019-07-30 at 09:09 -0700, Sean Whalen wrote:
> I just fixed this in the latest development version, 6.5.2, but I'm going to try to
> wait to do an actual release until I can verify that I have also fixed issue #102,
> which I suspect is related.
> In the meantime, you can upgrade to the latest dev release by running
> sudo -H pip3 install -U git+https://github.com/domainaware/parsedmarc.git
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
-- 

Worked for me - had to change the ini file from inbox.DMARC to "just" DMARC: Snippet from
file:
reports_folder = DMARC
archive_folder = DMARCArchive

was
reports_folder = inbox.DMARC
archive_folder = Inbox.DMARCArchive

Thanks, Sean

/Martin

**Comment by seanthegeek** [comment]
Great! I'll go ahead and release parsedmarc 6.5.2
